### PR TITLE
Remove time travel 1

### PR DESCRIPTION
**Pull request by jiaoew1991**
issue: https://github.com/milvus-io/milvus/issues/26070

### CODE REVIEW

File: internal/proxy/task_query.go [hunk 1]
```diff
@@ -361,16 +361,7 @@ func (t *queryTask) PreExecute(ctx context.Context) error {
...
-		t.TravelTimestamp = t.BeginTs()
-	} else {
-		t.TravelTimestamp = t.request.TravelTimestamp
-	}
-
-	err = validateTravelTimestamp(t.TravelTimestamp, t.BeginTs())
-	if err != nil {
-		return err
-	}
+	t.TravelTimestamp = t.BeginTs()
```
Comment (xiaofan-luan) [inline internal/proxy/task_query.go:364]: why to we change the segcore to remove t.TravelTimestamp?

File: internal/proxy/impl.go [hunk 2]
```diff
@@ -2664,8 +2662,7 @@ func (node *Proxy) Search(ctx context.Context, request *milvuspb.SearchRequest)
 		}
 	}()
 
-	log.Debug(
-		rpcReceived(method))
+	log.Debug(rpcReceived(method))
```
Comment (xiaofan-luan) [inline internal/proxy/impl.go:2665]: if we remove travel timestamp, we also remove the functionality of MVCC.
Any functionality relies on this?

File: internal/proxy/task_query.go (hunk 1 above)
Comment (xiaofan-luan) [reply internal/proxy/task_query.go:364]: but why use t.BeginTs as travel timestamp?
what's the difference between beginTs and endTs

Comment (jiaoew1991) [reply internal/proxy/task_query.go:364]: For task_query, beginTs and endTs are the same, both are t.Base.Timestamp, using the time of task enqueue.

In this PR, there are no changes to the settings. Only the "traveltimestamp" in the request has been removed,  is the default value is used as it was before when "traveltimestamp" was not set.

**Review by congqixia — APPROVED** [review]
/lgtm

**Pull request merged**